app/scripts/modules/azure/securityGroup/securityGroup.write.service.js

```diff
@@ -9,6 +9,8 @@ module.exports = angular
   .module('spinnaker.azure.securityGroup.write.service', [require('@uirouter/angularjs').default])
   .factory('azureSecurityGroupWriter', function() {
     function upsertSecurityGroup(securityGroup, application, descriptor, params = {}) {
+      params.securityGroupName = securityGroup.name;
```

Comment:
@christopherthielen wanted to ask you about this, not sure why azure has a different writer, just wanted to make sure I understood it correctly

Reply:
Not sure why.  This looks fine.